kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/servlets/AjaxPublicationsListServlet.java

```diff
@@ -232,7 +235,7 @@ public void doPost(HttpServletRequest req, HttpServletResponse res)
         writer.write(board.printAfter());
       } else if (NodePK.ROOT_NODE_ID.equals(kmeliaSC.getCurrentFolderId()) && kmeliaSC.
           getNbPublicationsOnRoot() != 0 && kmeliaSC.isTreeStructure() && !searchInProgress) {
-        displayLastPublications(kmeliaSC, resources, gef, writer);
+        displayLastPublications(kmeliaSC, resources, writer);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a "try/catch" block for "displayLastPublications". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1989)